package.json

```
@@ -1,13 +1,14 @@
 {
   "name": "react-onesignal",
-  "version": "1.7.0",
+  "version": "1.0.0",
```

Version should be 2.0.0-beta1

---

We can't downgrade the npm package version, so this should be

```suggestion
  "version": "2.0.0-beta1",
```